week5/feet_meters_test.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+
+class Test(unittest.TestCase):
```

Comment:
The name of the class is too generic, I suppose that it could be used by an other standard library. Prefer a more explicit name such as `FeetMeterTest`